OasisDEX/oasis-borrow
# Updated Take Profit banner

# [Updated Take Profit banner](https://app.shortcut.com/oazo-apps/story/6316/update-atp-banner-copy-graphic)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- updated ATP banner title, description and image
  
## How to test 🧪
  <Please explain how to test your changes>

- go to Optimization tab when ATP feature toggle enabled, ATP banner should be updated
